.github/workflows/release.yml

```diff
@@ -1,16 +1,37 @@
-name: release
+name: 'New Release'
+run-name: 'Release ${{ inputs.version_number }}'
+
 on:
-  push:
-    tags:
-      - 'v*'
-jobs: 
-  goreleaser:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: 'Version number (e.g., v1.0.0, v1.0.0-pre, v1.0.0-pre1)'
+        required: true
+      skip_tests:
+        description: 'Skip QA acceptance tests, define value to `true` to explicitly skip'
+
+jobs:
+    # QA acceptance tests are skipped for pre releases or when explicit input parameter is used
+  run-qa-acceptance-tests:
+    if: inputs.skip_tests != 'true' && !contains(inputs.version_number, 'pre')
```

Comment:
why do we always skip QA tests for pre-release? I see a scenario where we want to run the QA as part of the pre-release instead of the release. The pre-release will contain all the code changes except the changelog PR.

Reply:
did not want to slow down the process of generating the pre release, but I do see the value in detecting an issue in the prerelease vs seeing the issue in the final release. We can leave it to run by default, and have the option of skipping if you have already triggered manually for example.

Reply:
adjusted workflow and documentation.